src/cls/_ZPM/PackageManager/Developer/SystemRequirements.cls

```diff
@@ -0,0 +1,88 @@
+Class %ZPM.PackageManager.Developer.SystemRequirements Extends (%Persistent, %XML.Adaptor)
+{
+
+Property VersionString As %String(MAXLEN = 100, XMLNAME = "Version", XMLPROJECTION = "ATTRIBUTE");
+
+Property Version As %ZPM.PackageManager.Core.SemanticVersionExpression(XMLPROJECTION = "NONE");
+
+Method VersionStringSet(tValue) As %Status
+{
+	Set i%VersionString = tValue
+	Set tSC = ##class(%ZPM.PackageManager.Core.SemanticVersionExpression).FromString(tValue,.tExpression)
+	If $$$ISOK(tSC) {
+		Set ..Version = tExpression
+	}
+	Quit tSC
+}
+
+Property Product As %String(XMLPROJECTION = "ATTRIBUTE");
+
+Property Interoperability As %String(VALUELIST = ",enabled,disabled", XMLPROJECTION = "ATTRIBUTE");
+
+Method CheckRequirements() As %Status
+{
+	Set tSC1 = ..CheckVersion()
+	Set tSC2 = ..CheckInteroperability()
+	Return $$$ADDSC(tSC1, tSC2)
+}
+
+Method CheckVersion() As %Status
+{
+	Set tSystemMajor = $System.Version.GetMajor()
+	Set tSystemMinor = $System.Version.GetMinor()
+	If (..VersionString = "") { Return $$$OK } 
+	try {
+		Set tOperator = ..Version.Options.GetAt(1).Comparators.GetAt(1).Operator
+		Set tModuleMajor = ..Version.Options.GetAt(1).Comparators.GetAt(1).Major
+		Set tModuleMinor = ..Version.Options.GetAt(1).Comparators.GetAt(1).Minor
+		Set tVersionMatch = 0
+		Set tCommand = "(tSystemMajor,tModuleMajor,tSystemMinor,tModuleMinor,tVersionMatch) If ((tSystemMajor "_tOperator_" tModuleMajor) && (tSystemMinor "_tOperator_" tModuleMinor)) { Set tVersionMatch = 1 }"
+
+		Xecute (tCommand, tSystemMajor,tModuleMajor,tSystemMinor,tModuleMinor,.tVersionMatch)
```

Comment:
Xecute is good command in some cases, but not here. Should be replaced with $case, with a fixed list of operators.

Reply:
I changed to leverage existing version evaluation code, without Xecute.

We discussed GetFeature with Evgeny and he  said that it's not necessary now